substratevm/src/com.oracle.svm.jfr/src/com/oracle/svm/jfr/logging/JfrLogConfiguration.java

```diff
@@ -109,6 +111,22 @@ private static void verifySelections(JfrLogSelection[] selections) {
         result.put(LogTag.JFR_EVENT, EnumSet.of(JfrLogTag.JFR, JfrLogTag.EVENT));
         result.put(LogTag.JFR_SETTING, EnumSet.of(JfrLogTag.JFR, JfrLogTag.SETTING));
         result.put(LogTag.JFR_DCMD, EnumSet.of(JfrLogTag.JFR, JfrLogTag.DCMD));
+
+        // JDK17 support
+        if (JavaVersionUtil.JAVA_SPEC >= 17) {
+            try {
+                LogTag JFR_SYSTEM_STREAMING = Enum.valueOf(LogTag.class, "JFR_SYSTEM_STREAMING");
+                LogTag JFR_SYSTEM_THROTTLE = Enum.valueOf(LogTag.class, "JFR_SYSTEM_THROTTLE");
+                LogTag JFR_START = Enum.valueOf(LogTag.class, "JFR_START");
+
+                result.put(JFR_SYSTEM_STREAMING, EnumSet.of(JfrLogTag.JFR, JfrLogTag.SYSTEM, JfrLogTag.STREAMING));
+                result.put(JFR_SYSTEM_THROTTLE, EnumSet.of(JfrLogTag.JFR, JfrLogTag.SYSTEM, JfrLogTag.THROTTLE));
+                result.put(JFR_START, EnumSet.of(JfrLogTag.JFR, JfrLogTag.START));
+            } catch (IllegalArgumentException | NullPointerException e) {
+                throw VMError.shouldNotReachHere("Should be defined");
```

Comment:
`throw VMError.shouldNotReachHere(e);` or `throw VMError.shouldNotReachHere("Should be defined", e);`